triton-vm/src/stark.rs

```diff
@@ -192,10 +197,38 @@ impl Stark {
         prof_stop!(maybe_profiler, "Merkle tree");
         prof_stop!(maybe_profiler, "ext tables");
 
-        prof_start!(maybe_profiler, "quotient degree bounds");
-        let quotient_degree_bounds =
-            all_quotient_degree_bounds(self.interpolant_degree, master_base_table.padded_height);
-        prof_stop!(maybe_profiler, "quotient degree bounds");
+        prof_start!(maybe_profiler, "out-of-domain rows");
+        let trace_domain_generator = derive_domain_generator(padded_height.value());
+        let out_of_domain_value_curr_row = proof_stream.sample_scalars(1)[0];
+        let out_of_domain_value_next_row = trace_domain_generator * out_of_domain_value_curr_row;
```

Comment:
I would suggest to use "point" instead of "value" because "value" implies (to my ears at least) that it's the output of some evaluation. Here's it's the opposite: it's the *input*!